src/app/organization/employees/view-employee/view-employee.resolver.ts

```diff
@@ -0,0 +1,31 @@
+/** Angular Imports */
+import { Injectable } from '@angular/core';
+import { Resolve, ActivatedRouteSnapshot } from '@angular/router';
+
+/** rxjs Imports */
+import { Observable } from 'rxjs';
+
+/** Custom Services */
+import { OrganizationService } from 'app/organization/organization.service';
+
+/**
+ * Employee data resolver.
+ */
+@Injectable()
+export class ViewEmployeeResolver implements Resolve<Object> {
```

Comment:
```suggestion
export class EmployeeResolver implements Resolve<Object> {
```

Rename the file as well and update all the references as well.